pyomo/contrib/mindtpy/MindtPy.py

```diff
@@ -231,6 +231,21 @@ class MindtPySolver(object):
                     "slack variables here are used to deal with nonconvex MINLP",
         domain=bool
     ))
+    CONFIG.declare("continuous_var_bound", ConfigValue(
+        default=1e24,
```

Comment:
I think I would make this `sys.float_info.max` (usually 1.7976e+308), unless you have a reason to do something different.  1e24 is probably excessively restrictive.